armi/cases/case.py

```diff
@@ -563,38 +563,38 @@ def checkInputs(self):
         bool
             True if the inputs are all good, False otherwise
         """
+        runLog.header("=========== Settings Validation Checks ===========")
         with DirectoryChanger(self.cs.inputDirectory, dumpOnException=False):
             operatorClass = operators.getOperatorClassFromSettings(self.cs)
             inspector = operatorClass.inspector(self.cs)
             inspectorIssues = [query for query in inspector.queries if query]
-            if context.CURRENT_MODE == context.Mode.INTERACTIVE:
-                # if interactive, ask user to deal with settings issues
-                inspector.run()
-            else:
-                # when not interactive, just print out the info in the stdout
-                queryData = []
-                for i, query in enumerate(inspectorIssues, start=1):
-                    queryData.append(
-                        (
-                            i,
-                            textwrap.fill(
-                                query.statement, width=50, break_long_words=False
-                            ),
-                            textwrap.fill(
-                                query.question, width=50, break_long_words=False
-                            ),
-                        )
+
+            # Write out the settings validation issues that will be prompted for
+            # resolution if in an interactive session or forced to be resolved
+            # otherwise.
+            queryData = []
+            for i, query in enumerate(inspectorIssues, start=1):
+                queryData.append(
+                    (
+                        i,
+                        textwrap.fill(
+                            query.statement, width=50, break_long_words=False
+                        ),
+                        textwrap.fill(query.question, width=50, break_long_words=False),
                     )
+                )
 
-                if queryData and context.MPI_RANK == 0:
-                    runLog.header("=========== Settings Input Queries ===========")
-                    runLog.info(
-                        tabulate.tabulate(
-                            queryData,
-                            headers=["Number", "Statement", "Question"],
-                            tablefmt="armi",
-                        )
+            if queryData and context.MPI_RANK == 0:
+                runLog.info(
+                    tabulate.tabulate(
+                        queryData,
+                        headers=["Number", "Statement", "Question"],
+                        tablefmt="armi",
                     )
+                )
+            if context.CURRENT_MODE == context.Mode.INTERACTIVE:
+                # if interactive, ask user to deal with settings issues
+                inspector.run()
```

Comment:
This change provides some header in an interactive session to see the setting validators. I can pull out to a separate PR if needed, but this was a quality of life thing when I was testing.

Reply:
I can do this.

Reply:
Going to leave as is. This isnt a breaking change.